Crackmes/README.md

```diff
@@ -75,6 +75,33 @@ This app is compatible with Android 4.4 and up.
 - [Solution using Frida by Eduardo Novella](https://enovella.github.io/android/reverse/2017/05/20/android-owasp-crackmes-level-3.html "Solution by Eduardo Novella")
 - [Solution using patches by sh3llc0d3r](http://sh3llc0d3r.com/owasp-uncrackable-android-level3/ "Solution by sh3llc0d3r")
 
+## [UnCrackable App for Android Level 4: Radare2 Pay v1.0](Android/Level_04 "Android level 4")
+
+The Radare2 community always dreamed with its decentralized and free currency to allow r2 fans to make payments in places and transfer money between r2 users. A debug version has been  developed and it will be supported very soon in many stores and websites. Can you verify that this is cryptographically unbreakable?
```

Comment:
```suggestion
The Radare2 community always dreamed with its decentralized and free currency to allow r2 fans to make payments in places and transfer money between r2 users. A debug version has been developed and it will be supported very soon in many stores and websites. Can you verify that this is cryptographically unbreakable?
```